docs/sql/statements/create-role.rst

```diff
@@ -0,0 +1,31 @@
+.. highlight:: psql
+.. _ref-create-role:
+
+===============
+``CREATE ROLE``
+===============
+
+Create a new database role.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+Synopsis
+========
+
+.. code-block:: psql
+
+  CREATE ROLE roleName
+
+Description
```

Comment:
INHERIT | NOINHERIT
    | LOGIN | NOLOGIN

options will be added later?

Reply:
Since we don't support changing any of those after creation, I didn't want to add such info as it's only internal for now, and might confuse the users.

Reply:
Sorry, I was confused by checking parser changes, I missed that we reject those later.